src/legacy/core_plugins/input_control_vis/public/components/vis/list_control.tsx

```diff
@@ -143,6 +156,7 @@ class ListControlUi extends PureComponent<ListControlUiProps, ListControlUiState
         onChange={this.handleOnChange}
         singleSelection={!this.props.multiselect}
         data-test-subj={`listControlSelect${this.props.controlIndex}`}
+        inputRef={this.setTextInputRef}
```

Comment:
So, (almost?) all visualizations have `area-hidden=true` attribute. Their DOM structure is usually just a bunch of divs and an svg. Input control visualization, however, has this inner `input` field, which was focusable before and the test was failing:
<img width="765" alt="Screenshot 2020-02-21 at 14 07 08" src="https://user-images.githubusercontent.com/1937956/75040925-7eb6a680-54b3-11ea-974b-eac3363ca1b0.png">

Reply:
What do you mean that much of dashboard is wrapped in "area-hidden=true"? If the point here is that dashboard is basically useless when all the visualizations are hidden in accessibility mode, I completely agree. I think it doesn't make much sense to show the dashboard in the first place, but I kinda assumed you and Bhavya had already discussed that :D

Reply:
I hadn't done an a11y deep dive into the Dashboard page before so this is the first that I'm seeing it... I guess it's too large to address in this PR though. I'll focus on helping you merge this so that we can get the tests merged and I'll take a note to revisit it later. Might ping you about it later depending on how investigation goes.